Tasks/VSMobileCenterUpload/package.json

```diff
@@ -17,9 +17,8 @@
   },
   "homepage": "https://github.com/microsoft/vsts-tasks",
   "dependencies": {
-    "vsts-task-lib": "0.9.20",
+    "vsts-task-lib": "2.0.3-preview",
```

Comment:
Why use the 'preview' version?  IIRC, 2.0.5 is out already.  Do we want to specifically use a 'preview' version?